test/cmdlineargs.jl

```diff
@@ -2,6 +2,18 @@
 
 import Libdl
 
+macro with_timeout(arg)
+    quote
+        t = @schedule $(esc(arg))
+        @schedule begin
+            sleep(100)
+            istaskdone(t) && return
+            schedule(t, ErrorException("Timeout exceeded"); error=true)
```

Comment:
This is typically the wrong way to schedule timeouts. Please add a comment saying not to copy this macro